tests/testthat/test_get_axes.R

```diff
@@ -0,0 +1,14 @@
+context("get_axes")
+
+test_that("get x and y axes", {
+  p <- ggplot(iris, aes(Sepal.Length, Sepal.Width)) + geom_point()
```

Comment:
In general it's better to use made up data sets with only two or three rows, to speed up automated testing. E.g.:
``` r
df <- data.frame(x = 1:3, y = 1:3)
p <- ggplot(df, aes(x, y)) + geom_point()
```

Reply:
I had no idea about this! All switched.